.dev/performance/index.js

```diff
@@ -233,31 +233,31 @@ async function runPerformanceTests( branches, options ) {
 		results[ testSuite ] = {};
 		/** @type {Array<Record<string, WPPerformanceResults>>} */
 		const rawResults = [];
-		// Run the test three times
-		for ( let i = 0; i < 3; i++ ) {
+		// Run the test two times
```

Comment:
I am not opposed to running this two times as opposed to three but I am curious of the reason for this decision.

Reply:
@AnthonyLedesma Yeah, honestly I lowered it to 2 from 3 and it started to stabilize a bit more. I found that running it 3 times caused more failures, and increased the time to run. If you run these tests frequently enough, the performance test builds will fail with the same `Error ""` that we were running into before. The primary reason was just because it appeared the performance test runs were more stable.